elements/bulbs-video/elements/video-carousel.js

```diff
@@ -120,7 +120,11 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
 
     forEach.call(
       this.querySelectorAll('share-tools'),
-      (element) => element.setAttribute('share-url', this.state.shareUrl),
+      (element) => element.setAttribute('share-url', this.state.shareUrl)
```

Comment:
One annoyance is JS rejects a trailing comma in function invocation.

`console.log('hello', 'world',)` is a syntax error